src/components/AppHeader/AppHeader.css

```diff
@@ -1,6 +1,6 @@
-@import '../UI/theme';
+@import '../UI/theme.css';
```

Comment:
Don't import here

Reply:
Thanks, for some reason I had thought this was necessary to get the theme variables in this file. Reading the docs on it that's clearly not the case - not sure where I got that idea,